### ISSUE TITLE
add commit id to arc json 

### ISSUE THREAD

**Issue by kMutagene** [nfdi4plants/arc-export#2]
This is needed for DOI record versioning.

Might be an issue for arctr actually, where is this ideally implemented @HLWeil ?

**Comment by HLWeil** [comment]
Yup, this is definitely something for ARCtrl. Feel free to open an Issue.

I guess with "commit id" you mean the Hash of the last commit? The commit history is already part of the ARCtrl datamodel. It's just that the behaviour for getting this information is not yet implemented.

Feel free to open up an Issue there.
